src/System.Net.Http/tests/FunctionalTests/PlatformHandlerTest.cs

```diff
@@ -12,6 +13,8 @@ namespace System.Net.Http.Functional.Tests
 
     public class PlatformHandler_HttpClientHandler : HttpClientHandlerTestBase
     {
+        protected override bool UseSocketsHttpHandler => false;
```

Comment:
Woh, does this mean we've been testing the wrong handler for these tests this whole time?

Reply:
Apparently true :(

https://github.com/dotnet/corefx/blob/6322a406f88e854c5b827ab838f988f8992048f8/src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTestBase.cs#L14

Reply:
Although I have seen WINHTTP related exceptions before in the tests.  So, I think some of them are using "platform" handler.  Not sure when/how this broke.

Reply:
@wfurt, could you also quickly audit our other `PlatformHandler_` test classes to make sure there aren't others in this camp?

Reply:
I check and rest look ok.